source/multigrid/mg_transfer_internal.cc

```diff
@@ -117,6 +121,14 @@ namespace internal
                      numbers::artificial_subdomain_id))
                 continue;
 
+              if (has_active_cells_on_this_level == false)
```

Comment:
Does this make any difference at all? I guess you could also remove this if-clause?

Reply:
I removed the `if` clause and the variable as I can query the size only.